docs/source/upcoming_release_notes/362-bug_dict_invalid.rst

```diff
@@ -0,0 +1,23 @@
+362 bug_dict_invalid
+####################
+
+API Breaks
+----------
+- Client search methods will now return an `InvalidResult` if a container
```

Comment:
Docs suggestion: should we also document the previous behavior and rationale here for an API break?

> Previously, client search methods would raise a `KeyError` from the inciting exception. This `KeyError` will no longer be raised, and the inciting exception will be included as a data member in the `InvalidResult` object. This means downstream code that iterates over many `happi` entries doesn't need special error handling on big searches unless specifically the specific search item it needs is invalid.

Reply:
Possibly we should also add part of my blurb from my other review comment where I explain the API differences between `SearchResult` and `InvalidResult` (to the extent that comment is correct)

Reply:
Good point. It's good to be verbose for things like this

Reply:
updated in https://github.com/pcdshub/happi/pull/362/commits/7c3def5a24ab42e9fba99b1c26a6700f7946b2b1